src/storage/src/hummock/sstable/block.rs

```diff
@@ -139,10 +141,99 @@ impl RestartPoint {
     }
 }
 
-#[derive(Clone)]
+#[derive(Debug)]
+pub struct Hitmap<const N: usize> {
```

Comment:
`Box<[...]>` maybe better. Otherwise when N is large, it will be costly to move the `Hitmap`.